apps/web/src/server/applications/common/services/entities.service.js

```diff
@@ -33,6 +33,10 @@ export const getAllRegions = () => {
 	return get(`applications/region`);
 };
 
+export const getAllCaseStages = () => {
+	return get('applications/case-stage');
```

Comment:
this function doesn't handle errors. it's fine but you need some validation in the file above

Reply:
👍 Added a try-catch around the function consuming `getAllCaseStages`.